test/testcafe/spec/IdentifyUnknownUser_spec.js

```diff
@@ -34,6 +34,7 @@ test('should show messages callout for unknown user', async t => {
   await identityPage.clickNextButton();
   await t.expect(identityPage.getUnknownUserCalloutContent())
     .eql('Unable to sign in');
+  await t.expect(identityPage.hasUnknownUserErrorCallout()).eql(true);
```

Comment:
can you add a check that there is only one error message.. and not two.

Reply:
Done. https://github.com/okta/okta-signin-widget/pull/2196/files#diff-d85a7b24825cc1d5d0fb184084c66b2d1f7607c14befff9cbd8ac383a8edf716R48